deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+
+![Deploy Button](https://res.cloudinary.com/adeshina/image/upload/v1611604456/wdi3swnvoa2v2bhaaqpd.png)
+
+Next, click on the deploy button. Okteto automatically retrieves, deploys and starts your application.
+
+![Deployed Application](https://res.cloudinary.com/adeshina/image/upload/v1611604494/rscsvqxobd5dc5bt0baa.png)
+
+## Adding A New Route
+
+Let's update the application by adding a route for adding bulk recipes.
+
+In `app/api.py`, just below the POST route, add another route. Start by importing the `List` wrapper from typing before proceeding to adding the new route:
+
+```py
+from typing import List
+
+...
+
+@app.post("/recipes", tags=["Recipe"])
+def add_bulk_recipe(bulk_recipe: List[RecipeSchema] = Body(...)) -> dict:
+    for recipe in bulk_recipe:
+        recipe.id = len(recipes) + 1
+        recipes.append(recipe.dict())
+
+    return {            
+        "message": "Recipes added successfully"    
+    }
+
+```
+
+Test the new route from your terminal:
+
+```bash
+(venv)$ curl --request POST \
+  --url http://0.0.0.0:8080/recipes \
+  --header 'Content-Type: application/json' \
+  --data '[
+	{
+		"name": "Donuts",
+		"ingredients": [
+			"Flour",
+			"Milk",
+			"Sugar",
+			"Vegetable Oil"
+		]
+	},
+	{
+		"name": "Creamy Donuts",
+		"ingredients": [
+			"Flour",
+			"Milk",
+			"Sugar",
+			"Vegetable Oil",
+			"Irish whipped cream"
+		]
+	}
+]'
+```
+
+You will get a response:
+
+```json
+{
+  "message": "Recipes added successfully."
+}
+```
+
+Commit the new change in the `app/api.py` file into a new branch:
+
+```bash
+(venv)$ git checkout -b bulk-recipes && git add app/api.py
+(venv)$ git commit -m "Add bulk recipes"
+(venv)$ git push -u origin bulk-recipes
+```
+
+Verify that the new branch has been created in your GitHub repo.
+
+## Redeploying
+
+Once again, login into your Okteto dashboard and click on the redeploy button:
+
+![Redeploy Button](https://res.cloudinary.com/adeshina/image/upload/v1611604547/vhenrnydsenkb6a0xd3e.jpg)
+
+Change the branch from `main` to `bulk-recipes` and deploy:
+
+![Image 5](https://res.cloudinary.com/adeshina/image/upload/v1611604639/knpe6yiz4alvplgniszb.png)
+
+The branch on which the redeployment can be seen:
+
+![bulk-recipes branch](https://res.cloudinary.com/adeshina/image/upload/v1611604587/isccfvfifunqfmnevtp8.jpg)
+
+You can go on to test the new route by replacing `localhost` from previous requests to the live application url.
+
+
+## Conclusion
+
+In this article, you deployed an application directly from the Okteto dashboard and redeployed another new branch application version. The deployment was done with minor changes to the Okteto stack file, demonstrating the adaptability of the Okteto stacks. 
+
+Additionally, Okteto can perform the deployment from different branches from the CLI tool. You can find the code from this article on [GitHub]
+(https://github.com/Okteto/fastapi-crud/tree/bulk-recipes)
```

Comment:
I think it would be better if we merge this two paragraphs into one, and use the second one to remidn the user that okteto is free, and tell them where to sign up for a new account.

Reply:
sounds nice. I'll do that.